nodeup/pkg/model/containerd.go

```diff
@@ -107,14 +102,6 @@ var containerdVersions = []packageVersion{
 		Version:        "1.2.10",
 		Source:         "https://download.docker.com/linux/centos/7/x86_64/stable/Packages/containerd.io-1.2.10-3.2.el7.x86_64.rpm",
 		Hash:           "f6447e84479df3a58ce04a3da87ccc384663493b",
-		ExtraPackages: map[string]packageInfo{
-			"container-selinux": {
```

Comment:
I guess the problem here is that container-selinux might be tied to the behaviour of a particular version of containerd.  i.e. a new version of containerd introduces a new required permission (?)

Reply:
My assumption was that newer container-selinux will only add permissions, which would make it backwards compatible with any containerd or docker version. In a regular install. where one has a valid subscription, any package upgrade would grab the latest version.